local-cli/runIOS/runIOS.js

```diff
@@ -129,7 +133,15 @@ function buildProject(xcodeProject, udid, scheme) {
     '-derivedDataPath', 'build',
   ];
   console.log(`Building using "xcodebuild ${xcodebuildArgs.join(' ')}"`);
-  child_process.spawnSync('xcodebuild', xcodebuildArgs, {stdio: 'inherit'});
+  var buildProcess = child_process.spawnSync('xcodebuild', xcodebuildArgs, {stdio: 'pipe'});
+  var buildOutput = buildProcess.stdout.toString();
+  console.log(buildOutput);
```

Comment:
Doing `console.log` like this means we will buffer all output and it will seem like the process is stuck while xcode is building. Can you stream it to the console while also scanning it?